### DISCUSSION TITLE
Meeting Notes for GSoC (6/10/2024)

### DISCUSSION THREAD

**Discussion by JackyZzZz** [theochem/Selector#211] [General]
# Meeting Notes

**Date:** 6/10/2024
**Participants:** @FanwangM   @JackyZzZz

## Meeting Overview

1. **Deployment on HuggingFace**
   - Prioritize setting this up so it is publicly accessible for feedback.
   - For simplicity, manually push the source code to host it for now.

2. **Frequency of Opening Pull Requests**
   - Split tasks into smaller PRs for easier code review.
   - Treat PRs as drafts for feedback without waiting. Merge once everything is settled.

## Next Steps

**To-Do Before Next Meeting:**
- Set up the deployment pipeline on Hugging Face.
- Continue working on the interface for other algorithms.

**Comment by FanwangM** [comment]
Thanks for the notes! Keep the momentum, please.
